doc/docker.md

```diff
@@ -0,0 +1,72 @@
+# Docker
+
+To have the smallest possible changes between a deployment in pure
+Ansible and a container based one, docker images are built from
+ansible roles.
+
+The `role2docker` script in the `bin` directory creates the docker
+image for a specific role. For example: `./bin/role2docker wazo_message_bus`.
+
+## Settings
+
+A role that can be used to create a container must have a `docker` directory.
+
+This `docker` directory can contain the following files:
+
+`pre`: executable to run before installing the role on the
+container. This is usually used to install dependencies from galaxy.
+
+`post`: executable run after installing the role.
+
+`init`: executable that is coped in `/init` and used as the entry point.
+
+`cmd`: entry point to inject in the container.
```

Comment:
Same as `init` above, typo ?  I guess you meant

```suggestion
`cmd`: command to be passed down to the entrypoint.
```

Reply:
ok I clarified the doc.